elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -32,17 +32,17 @@ class PfeContentSet extends PFElement {
   get settings() {
     let settings = {};
     const variant = this.getAttribute("pfe-variant");
+    const theme = this.getAttribute("on");
 
-    if (variant === "primary") {
+    if (variant === "wind") {
       settings.variant = variant;
-      settings.color = "striped";
-    } else if (variant === "secondary") {
+    } else if (variant === "earth") {
```

Comment:
Is earth the default or is there default, wind, and earth?

Reply:
Wind is the default. I'll switch this around...
```
if (variant === "earth") {
   do this;
} else {
  do this;
}

Reply:
@castastrophe I have made this change ^